packages/dds/merge-tree/package.json

```diff
@@ -14,7 +14,7 @@
     "build": "concurrently npm:build:compile npm:lint && npm run build:docs",
     "build:commonjs": "npm run tsc && npm run build:test",
     "build:compile": "concurrently npm:build:commonjs npm:build:esnext",
-    "build:docs": "echo \"Skipping @fluidframework/merge-tree; see https://github.com/microsoft/FluidFramework/issues/2543\"",
+    "build:docs": "api-extractor run --local --typescript-compiler-folder ../../../node_modules/typescript && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../_api-extractor-temp/",
```

Comment:
I think the CI failure is because there's no `ci:build:docs` script defined here.

Reply:
Was also getting flaky implicit typing on `canAppend`, added some explicit types to avoid confusion.